### ISSUE TITLE
Service Delivery for CE

### ISSUE THREAD

**Issue by timpitcher2** [MicrosoftDocs/dynamics-365-unified-operations-public#3611]
Hi! Is there an equivalent page for Customer Engagement applications?!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: df0b4c4c-1831-e430-f6d6-2d5056c75e26
* Version Independent ID: f3a731aa-5836-c509-08ba-7a27318bef34
* Content: [Service description for Finance and Operations apps - Finance & Operations | Dynamics 365](https://docs.microsoft.com/en-us/dynamics365/fin-ops-core/fin-ops/get-started/service-description?toc=/dynamics365/finance/toc.json)
* Content Source: [articles/fin-ops-core/fin-ops/get-started/service-description.md](https://github.com/MicrosoftDocs/Dynamics-365-Unified-Operations-Public/blob/main/articles/fin-ops-core/fin-ops/get-started/service-description.md)
* Service: **dynamics-365-cross-app**
* GitHub Login: @tomhig
* Microsoft Alias: **whigginb**

**Comment by tomhig** [comment]
No, there isn't. This was an outdated White paper from the original ERP service released in 2016. Content was overhauled and brought on to Docs. The implementation process is quite different from the Customer Engagement apps.

Sent from Surface Duo
________________________________
From: timpitcher2 ***@***.***>
Sent: Friday, October 8, 2021 6:45:55 AM
To: MicrosoftDocs/dynamics-365-unified-operations-public ***@***.***>
Cc: Tom Higginbotham ***@***.***>; Mention ***@***.***>
Subject: [MicrosoftDocs/dynamics-365-unified-operations-public] Service Delivery for CE (#3611)


Hi! Is there an equivalent page for Customer Engagement applications?!

________________________________
Document Details

⚠ Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.

  *   ID: df0b4c4c-1831-e430-f6d6-2d5056c75e26
  *   Version Independent ID: f3a731aa-5836-c509-08ba-7a27318bef34
  *   Content: Service description for Finance and Operations apps - Finance & Operations | Dynamics 365<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fdynamics365%2Ffin-ops-core%2Ffin-ops%2Fget-started%2Fservice-description%3Ftoc%3D%2Fdynamics365%2Ffinance%2Ftoc.json&data=04%7C01%7CTom.Higginbotham%40microsoft.com%7C871ba6518e884603f7eb08d98a48cea0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637692867587669834%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=kIJOyMnCW08PuaIKM2gtXwYRSZ7qc0dVloZQH2welhA%3D&reserved=0>
  *   Content Source: articles/fin-ops-core/fin-ops/get-started/service-description.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FDynamics-365-Unified-Operations-Public%2Fblob%2Fmain%2Farticles%2Ffin-ops-core%2Ffin-ops%2Fget-started%2Fservice-description.md&data=04%7C01%7CTom.Higginbotham%40microsoft.com%7C871ba6518e884603f7eb08d98a48cea0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637692867587679783%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ETYvDVPTFUVh734WN6q6nvSHrJKf23aDvoYmk%2F0IdfU%3D&reserved=0>
  *   Service: dynamics-365-cross-app
  *   GitHub Login: @tomhig<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ftomhig&data=04%7C01%7CTom.Higginbotham%40microsoft.com%7C871ba6518e884603f7eb08d98a48cea0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637692867587679783%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=HCxHLsjinsdLyDwGWA3n3wECCU%2FmWKsmE%2FpLFYB54d8%3D&reserved=0>
  *   Microsoft Alias: whigginb

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fdynamics-365-unified-operations-public%2Fissues%2F3611&data=04%7C01%7CTom.Higginbotham%40microsoft.com%7C871ba6518e884603f7eb08d98a48cea0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637692867587689739%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=k2yZmMALHuUbc4V6KrWZG9mOqPjM4v5DV9462gTqAvw%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAEVB5CQCQK35P2APWCHYTADUF3DWHANCNFSM5FTKPUEQ&data=04%7C01%7CTom.Higginbotham%40microsoft.com%7C871ba6518e884603f7eb08d98a48cea0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637692867587689739%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=a2C5VVi93BGwLUhJKZfB%2BnNbpNVHnKmqJLvwGT%2FYLy4%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7CTom.Higginbotham%40microsoft.com%7C871ba6518e884603f7eb08d98a48cea0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637692867587699697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=UgCohu4lm%2BDsjRcxCAy54hDIxIGjOYbdwe1CtnSCt%2FA%3D&reserved=0> or Android<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7CTom.Higginbotham%40microsoft.com%7C871ba6518e884603f7eb08d98a48cea0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637692867587699697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=xiFM%2BKHU5HzoVX3KMtCEvPsr3Zm7YzwkZbqmdW1R8Zo%3D&reserved=0>.
